_content/gateways/registration.md

```diff
@@ -4,47 +4,49 @@ title: Gateway Registration
 
 # Gateway Registration
 
-We are transitioning to a new way to register and manage your gateways. Please read this guide carefully to understand how it currently works and what will change.
+The are mainly two types of packet forwarders that can be running on your gateway, namely [TTN Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/master) (formerly known as the Gateway Connector) or [UDP Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/legacy) (also known as the Semtech Packet Forwarder or Poly Packet Forwarder). 
```

Comment:
The TTN Packet Forwarder is new, it didn't have an old name before

Note that only like 10 users ever used the "gateway connector" option; this is not a well known thing